skipper.yaml

```diff
@@ -1,5 +1,5 @@
 registry: quay.io
-build-container-image: assisted-test-infra
+build-container-image: assisted-test-infra-build
```

Comment:
why, they are not the same? What if I want to work with terraform?

Reply:
`assisted-test-infra-build` is `assisted-test-infra` + another layer with assisted-service-client. We currently can't test the building of that image in the CI `images` part as we don't have assisted-service-code, so I splited the images